skimage/filters/rank/__init__.py

```diff
@@ -22,6 +22,7 @@
            'subtract_mean',
            'subtract_mean_percentile',
            'median',
+           'median_float',
```

Comment:
Can't ``median`` detect the dtype and respond by calling the correct underlying implementation?